tools/cli/commands.js

```diff
@@ -1731,11 +1731,14 @@ main.registerCommand({
   // Doesn't actually take an argument, but we want to print an custom
   // error message if they try to pass one.
   maxArgs: 1,
+  options: {
+    db: { type: Boolean },
+  },
   requiresApp: true,
   catalogRefresh: new catalog.Refresh.Never()
 }, async function (options) {
   if (options.args.length !== 0) {
-    Console.error("meteor reset only affects the locally stored database.");
+    Console.error("meteor reset might delete the local database using the --db option.");
```

Comment:
This warning sounds strange, will this be only shown if the user adds `--db` or is it easy to make it so? Perhaps we can keep the original message then...

Reply:
This appears when you use `meteor reset XXX` being XXX anything that is not an option.

![image](https://github.com/meteor/meteor/assets/2581993/c48eda7f-88b4-4279-9c4d-f5b40a195233)

I may rephrase like this:

`'meteor reset' command only affects the local project cache. Use the --db option to remove the local database.`

or how would you suggest?

Reply:
I think that makes sense